vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -127,10 +132,15 @@ NS_SWIFT_NAME(BraveAds)
 /// Report that a tab with a given id was closed by the user
 - (void)reportTabClosedWithTabId:(NSInteger)tabId NS_SWIFT_NAME(reportTabClosed(tabId:));
 
-/// Report that a notification event type was triggered for a given id
-- (void)reportAdNotificationEvent:(NSString *)notificationUuid
+/// Report that an ad notification event type was triggered for a given id
+- (void)reportAdNotificationEvent:(NSString *)uuid
                         eventType:(BATAdNotificationEventType)eventType;
 
+/// Report that a new tab page ad event type was triggered for a given id
+- (void)reportNewTabPageAdEvent:(NSString *)uuid
+           creative_instance_id:(NSString *)creative_instance_id
```

Comment:
Small Obj-C nit: `creative_instance_id` → `creativeInstanceId`

(+ the implementation in BATBraveAds.mm)

Reply:
Fixed